[Bug] Exception thrown when 'gradlew checkstyleMain' run

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.


### Environment

Windows

### EventMesh version

master

### What happened

When run `gradlew checkstyleMain` to check the project checkstyle. exception will be thrown.
```
Execution failed for task ':eventmesh-common:checkstyleMain'.
> A failure occurred while executing org.gradle.api.plugins.quality.internal.CheckstyleAction
   > An unexpected error occurred configuring and executing Checkstyle.
      > Unable to create Root Module: config {/home/runner/work/eventmesh/eventmesh/style/checkStyle.xml}, classpath {null}.
```

### How to reproduce

Run `./gradlew checkstyleMain`. If you want get detail run `./gradlew checkstyleMain --stacktrace`

(If you do checking by click the button of the CheckStyle plugin in IDE, it will execute correctly.)

### Debug logs

```
Execution failed for task ':eventmesh-common:checkstyleMain'.
> A failure occurred while executing org.gradle.api.plugins.quality.internal.CheckstyleAction
   > An unexpected error occurred configuring and executing Checkstyle.
      > Unable to create Root Module: config {/home/runner/work/eventmesh/eventmesh/style/checkStyle.xml}, classpath {null}.

Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to find: /checkstyle-header1.txt
        at com.puppycrawl.tools.checkstyle.utils.CommonUtil.getResourceFromClassPath(CommonUtil.java:457)
        at com.puppycrawl.tools.checkstyle.utils.CommonUtil.getFilepathOrClasspathUri(CommonUtil.java:434)
        at com.puppycrawl.tools.checkstyle.utils.CommonUtil.getUriByFilename(CommonUtil.java:385)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean$UriConverter.convert(AutomaticBean.java:341)
        ... 47 more
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

--- a/build.gradle
+++ b/build.gradle
@@ -81,6 +81,7 @@ allprojects {
         showViolations = true
         maxWarnings = 0
         configFile = new File("${rootDir}/style/checkStyle.xml")
+        configDirectory = file("${rootDir}/style")
     }
 
     checkstyleMain.exclude '**/org/apache/eventmesh/client/grpc/protos**'

--- a/style/checkStyle.xml
+++ b/style/checkStyle.xml
@@ -27,15 +27,15 @@
   <property name="fileExtensions" value="java, properties, xml, gradle, env, sh"/>
   <module name="SuppressWarningsFilter" />
   <module name="Header">
-    <property name="headerFile" value="/checkstyle-header1.txt"/>
+    <property name="headerFile" value="${config_loc}/checkstyle-header1.txt"/>
     <property name="fileExtensions" value="java, gradle"/>
   </module>
   <module name="Header">
-    <property name="headerFile" value="/checkstyle-header2.txt"/>
+    <property name="headerFile" value="${config_loc}/checkstyle-header2.txt"/>
     <property name="fileExtensions" value="properties, env"/>
   </module>
   <module name="Header">
-    <property name="headerFile" value="/checkstyle-header3.txt"/>
+    <property name="headerFile" value="${config_loc}/checkstyle-header3.txt"/>
     <property name="fileExtensions" value="sh"/>
   </module>
   <module name="RegexpSingleline">

